resources/particle_frag.glsl

```diff
@@ -0,0 +1,66 @@
+#version 330 core
```

Comment:
I forgot to get rid of it particle frag, but Austin suggested we use your diffuse frag, so it's a copy of that

Reply:
This shader is never used:
https://github.com/jaafersheriff/BattleRoyale/blob/dev_Particles/src/App/main.cpp#L252

Reply:
Okay, do you or do you not want me using diffuse_frag? If so, I'll remove particle_frag from the repo, but in that case how are you going to complain about diffuse_frag having so much logic? If not, I'll just use particle_frag and remove the logic. Just tell me.

Reply:
If these particles are bound to a mesh then they'll implicitly need a decent amount of logic in their GLSL shaders. diffuse_frag includes texture mapping, cell shading, silhouettes, and now shadows. I just don't know if all of that is necessary for particles - especially because given how shadows is implemented, particles will be shadow receivers but not shadow casters. 
Are the particles being sorted in any way before they're rendered? If yes, then we can play with the render order so that fragments aren't repeatedly recomputed and overwritten. If no, then let's just move forward with this PR and have ParticleShader share diffuse_frag. If it doesn't look good or if there's a significant performance hit we can copy diffuse_frag into a new particle_frag and cut down on the unnecessary logic.